### DISCUSSION TITLE
URLs containing the at character (@) are censored?

### DISCUSSION THREAD

**Discussion by pabs3** [community/community#12121] [Projects and Issues]
I included a URL containing an at character (@) in a GitHub issue
comment and the URL was censored. Presumably this is some sort of email
hiding system? I note that both Medium and TikTok use URLs that contain
the character in question. While TikTok videos seem unlikely to be
linked often (except maybe in youtube-dl/yt-dlp issues), Medium
articles about git and GitHub are probably linked sometimes. It would
be great if you could make the email hiding avoid breaking URLs.

https://medium.com/@zeotapstories/the-zeotech-series-git-workflow-rebase-vs-merge-93cf0866ed35
https://github.com/microsoft/DiskANN/issues/34#issuecomment-1051345676

**Comment by ajsirota** [comment]
It seems like this happens when you use email replies, and GitHub thinks it's an email address and privatizes it.
